test/admin/push.test.js

```diff
@@ -0,0 +1,111 @@
+const StitchMongoFixture = require('../fixtures/stitch_mongo_fixture');
+
+import { getAuthenticatedClient } from '../testutil';
+
+describe('Push Notifications V2', () => {
+  let test = new StitchMongoFixture();
+  let pushNotifications;
+  let app;
+  let apps;
+  beforeAll(() => test.setup({ createApp: false }));
+  afterAll(() => test.teardown());
+  beforeEach(async () => {
+    let adminClient = await getAuthenticatedClient(test.userData.apiKey.key);
+    test.groupId = test.userData.group.groupId;
+    apps = await adminClient.v2().apps(test.groupId);
+    app = await apps.create({ name: 'testname' });
+    pushNotifications = adminClient
+      .v2()
+      .apps(test.groupId)
+      .app(app._id)
+      .pushNotifications();
+  });
+  afterEach(async () => {
+    await apps.app(app._id).remove();
+  });
+
+  const DRAFT_TYPE = 'draft';
+  const SENT_TYPE = 'sent';
+  const testNotification = {
+    type: DRAFT_TYPE,
+    message: 'this is a test notification.',
+    label: 'test',
+    topic: 'v2'
+  };
+
+  it('listing draft push notifications should return empty list', async () => {
+    expect.assertions(1);
```

Comment:
You can actually omit `expect.assertions` in any tests where you're not using any branching that would result in a test assertion to get skipped over (like an if/else, try/catch, or .then/.catch). I don't think we need them in this file.